ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -85,36 +125,117 @@ pipeline {
             options {
                 timeout(time: 100, unit: 'MINUTES')
             }
-            steps {
-                script {
-                    parallel(
-                        "K8s-1.11":{
-                            sh 'cd ${TESTDIR}; K8S_VERSION=1.11 ginkgo --focus=" K8s*" -v --failFast=${FAILFAST}'
-                        },
-                        "K8s-1.12":{
-                            sh 'cd ${TESTDIR}; K8S_VERSION=1.12 ginkgo --focus=" K8s*" -v --failFast=${FAILFAST}'
-                        },
-                        failFast: "${FAILFAST}".toBoolean()
-                    )
+            failFast true
```

Comment:
This failfast should be with the label no?

Reply:
I couldn't get the label to work with this format :/